usaspending_api/recipient/v2/views/recipients.py

```diff
@@ -169,11 +170,26 @@ def extract_business_categories(recipient_name, recipient_duns):
         Returns:
             list of business categories
     """
-    if recipient_name in SPECIAL_CASES and recipient_duns is None:
-        return []
-    qs_business_cat = LegalEntity.objects.filter(recipient_name=recipient_name, recipient_unique_id=recipient_duns)\
+    business_categories = set()
+    if recipient_name in SPECIAL_CASES:
+        return list(business_categories)
+
+    # Go through DUNS first
+    d_business_cat = DUNS.objects.filter(legal_business_name=recipient_name, awardee_or_recipient_uniqu=recipient_duns)\
```

Comment:
RecipientLookup should now have business types from the DUNS table, you could then use `recipient_hash` for the search